code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -405,6 +414,14 @@
 	if(isAdminLevel(src.z))
 		return
 
+	for(var/obj/machinery/anti_bluespace/AB in range(8, src))
+		if(AB.stat & (NOPOWER | BROKEN))
+			continue
+		else
+			AB.use_power_oneoff(AB.active_power_usage)
+			to_chat(src, SPAN_WARNING("An unknown force prevents you from stepping into the Veil!"))
+			return
+
```

Comment:
Part of the purpose of bluespace inhibitors is to prevent antags who can teleport rushing stuff like the captain's spare or the armory 10 minutes into the round, and vampires frequently will do this, particularly in rounds with no captain to safeguard it. I think that preventing vampires from doing this is reasonable, and I can't think of a better solution than using what's already there.